package.json

```diff
@@ -47,6 +48,7 @@
     "@types/node": "18.6.2",
     "@typescript-eslint/eslint-plugin": "5.36.2",
     "@typescript-eslint/parser": "5.36.2",
+    "cross-var": "^1.1.0",
```

Comment:
what's cross-var?

Reply:
Just a minor dependency that will handle letting the docs build on Windows as well as Linux